lms/djangoapps/course_api/tests/test_api.py

```diff
@@ -209,6 +220,28 @@ def test_filter(self):
             assert {course.id for course in filtered_courses} == {course.id for course in expected_courses},\
                 f'testing course_api.api.list_courses with filter_={filter_}'
 
+    def test_permissions(self):
+
+        # Create a second course to be filtered out of queries.
+        self.create_course(course='i-dont-want-to-see-you-course')
```

Comment:
Minor: Although this is an absolutely perfect test, how about using different wording for the course name?

```suggestion
        self.create_course(course='not-included-course')
```